packages/front/svelte/views/auth/signup/SignupForm.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+    import SignupFormController from "./SignupForm.controller";
+
+    const ctrl = new SignupFormController();
+    const { stepId, currentStepData, step, nextStep, stepsStateById } = ctrl;
+</script>
+
+<h1>Titre exemple du formulaire</h1>
+
+<div class="fr-stepper">
+    <h2 class="fr-stepper__title">
+        <span class="fr-stepper__state">Étape {$step.index} sur {ctrl.finalStepIndex}</span>
+        {$step.title || ""}
+    </h2>
+    <div class="fr-stepper__steps" data-fr-current-step={$step.index} data-fr-steps={ctrl.finalStepIndex} />
+    {#if $nextStep}
+        <p class="fr-stepper__details">
+            <span class="fr-text--bold">Étape suivante :</span>
+            {$nextStep.title || `étape ${$nextStep.index}`}
+        </p>
+    {/if}
+</div>
+
+{#key $stepId}
```

Comment:
ça me semble complexe et en plus tu n'as pas prévu d'avoir un seul <form> dans ce fichier justement ?

Je lis ici l'utilisation de "megaform" en utilisant des fieldset pour gérer les étapes : https://lilyreile.medium.com/the-megaform-pattern-for-multi-step-forms-8040ff0c2e7a

Reply:
Je n'ai pas compris la phrase
> ça me semble complexe et en plus tu n'as pas prévu d'avoir un seul
> dans ce fichier justement ?

Mais par rapport à gérer les étapes via un fieldset ça ne va pas vu qu'on veut submit à chaque étape

Reply:
*un seul `<form>`

Reply:
Et je n'ai pas souvenir qu'on ait acté qu'on sauvegardait à chaque étape. Pour moi c'était encore en réflexion vu qu'on aurait pour le début eu d'une seule étape

Reply:
Alix m'a confirmé qu'il faudra savoir où les gens s'arrêtent donc on va devoir le faire

Reply:
Hmm perso je trouve ça overkill et time consuming. On devrait rediscuter de ça et de l'intérêt de savoir ou les gens s'arrêtent. Il faut d'abord prendre du temps et voir si beaucoup de gens n'activent pas leur compte avant de se demander si c'est nécessaire de faire ce dev.

@VSerain tu en penses quoi ?

Pour moi c'est à faire bien plus tard, si on le fait tout court

Reply:
Je veux bien qu'on en redicute en vrai